### ISSUE TITLE
Unable to start v7.2.0 due to Type load failure in Microsoft.PowerShell.PSReadLine.Polyfiller

### ISSUE THREAD

**Issue by shufton** [PowerShell/PSReadLine#3019]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19042
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![pwsh](https://user-images.githubusercontent.com/4588664/141430680-31da4ec2-9a73-497e-89ad-a55df4261804.png)


### Environment data

```console
Unable to execute this as pwsh fails to start.
OS Version	10.0.19042 Build 19042
```

### Steps to reproduce

ran `choco upgrade all -y` from command prompt to upgrade pwsh, no other instances running
Have uninstalled/installed from choco, and from programs and features to try and fix, but continues to crash
Attempt to start pwsh from start menu, Windows terminal, or from existing command prompt.

Additional info:
Have disabled all extra personal profile steps/functions
Same behaviour when running from root/non-root account

### Expected behavior

Successful app start

### Actual behavior

Crashes almost immediately, can guess from stack dump and screenshot that it seems to be once it starts waiting for input.

**Comment by shufton** [comment]
As some additional information, this has been reproduced on 2 separate machines (work and home) and downgrade to v7.1.5 doesn't fix it. Also reinstalled directly from msi download rather than choco.

**Comment by shufton** [comment]
Have found a downgrade path that works for me, uninstall pwsh 7.2.0, then from cmd.exe
```cmd
choco install pwsh -y --version 7.1.5
pwsh -Command "Install-Module PSReadLine -Force"
pwsh -Command "Uninstall-Module PSReadLine -RequiredVersion 2.2.0-beta1 -AllowPrerelease"
```

**Comment by iSazonov** [comment]
Please update to latest Beta4 version.

**Comment by leo2d** [comment]
Update the PSReadLine version worked for me.  One way for doing this is to open a new cmd window and go to your PowerShell 7 installation folder(on windows 10/11 is C:\Program Files\PowerShell\7 )  and then run 
`pwsh.exe -noprofile -command "Install-Module PSReadLine -Force -SkipPublisherCheck -AllowPrerelease"`

**Comment by shufton** [comment]
Just tested and yes Beta4 fixes it, so closing this now.

Many thanks for your help, it is greatly appreciated.

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
